### DISCUSSION TITLE
Social media preview for organizations

### DISCUSSION THREAD

**Discussion by robertlove** [community/community#31814] [Other Feature Feedback, Questions, & Ideas]
In the same way you can add a [social media preview](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview) image to a repository, the same functionality should also be available to organizations. Currently the organization's profile picture is used as the social media preview image - which is not great.

**Comment by jfullstackdev** [comment]
I'm not very sure but this is one of my Orgs

![image](https://user-images.githubusercontent.com/47092464/188262503-ee2ec25d-51e5-4181-b626-31c8f491f0e0.png)

there is that Social Preview on the repo's setting

**Comment by dimapasko** [comment]
Yeah, Social media preview option should be visible only for public repos.

It takes so much space on General tab.

**Reply by ackwolver335** [reply]
- where does it is shown or from which social media we can see it ?

**Comment by nicosammito** [comment]
This would be rly great 👍🏻

**Comment by robertlove** [comment]
I'd like to clarify this request as the first two responders appear confused.

This is a request for the Social preview feature to be implemented for **organizations**.

This feature is already available for **repositories** - so there is no need to discuss repositories, or post images of your repository settings.

- This is a **repository**: https://github.com/microsoft/TypeScript
- This is an **organization**: https://github.com/microsoft

Note the repository is using the **Social preview** feature found on the repository settings page. This is how it will be rendered: https://metatags.io/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2FTypeScript

Note the organization simply defaults to the Profile picture setting found on the organization settings page. This is how it will be rendered:: https://metatags.io/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft

This is not great as, in this day and age, an organization's GitHub page is a pivotal part of their brand and is shared widely across all media channels. Therefore, the ability to have your GitHub organization page appear as professional as possible is of great value.

**Reply by nicosammito** [reply]
Would be good to have the option to upload either a brand picture or a social media preview

**Reply by eclecticc** [reply]
For anyone wondering, updating the social image in the organization's .github repository doesn't seem to impact the social image that is used.  The social image remains the profile picture in the organization settings.
